routing/osrm_router.cpp

```diff
@@ -219,20 +227,43 @@ class Point2PhantomNode
     ft.ParseGeometry(FeatureType::BEST_GEOMETRY);
 
     // node.m_seg always forward ordered (m_pointStart < m_pointEnd)
-    distance -= MercatorBounds::DistanceOnEarth(ft.GetPoint(forward ? seg.m_pointEnd : seg.m_pointStart), segPt);
+    distance -= MercatorBounds::DistanceOnEarth(ft.GetPoint(offsetToRight ? seg.m_pointEnd : seg.m_pointStart), segPt);
+
+    // Offset is measured in decades of seconds. We don't know about speed restrictions on the road.
+    // So we find it through a whole edge weight.
+
+    // Whole node distance in meters.
+    double fullDist = 0.0;
```

Comment:
It's up to you, but how about to rename fullDist to fullDistInMeters or fullDistM? The same for distance.